src/common/mipmap_cache.c

```diff
@@ -77,11 +79,12 @@ struct dt_mipmap_buffer_dsc
   uint32_t pre_monochrome_demosaiced;
   dt_colorspaces_color_profile_type_t color_space;
   /* NB: sizeof must be a multiple of 4*sizeof(float) */
-} __attribute__((packed, aligned(16)));
+} __attribute__((packed, aligned(4 * sizeof(float))));
```

Comment:
whatever. SSE has 16-byte. who knows whether the logic is really 4*sizeof(float). there are also 16*sizeof(uint8_t) operations, no?

but would any of this need to change for omp simd and avx?